docs/test/bdd_instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instruction to run bdd tests
+
+## Demonstration
+- Two agents establish connection using did exchange protocol sdk
```

Comment:
I feel it would be helpful if we can map these list items to the "name of the test"/"bdd scenario name", so that it would be easy to run command mentioned in [line 23](https://github.com/hyperledger/aries-framework-go/pull/773/files#diff-538c2384a28ae378045d604a00b3d1afR23).

Reply:
done